pwa/src/templates/documentation/usageDocumentation/UsageDocumentationTemplate.tsx

```diff
@@ -128,26 +129,22 @@ export const UsageDocumentationTemplate: React.FC = () => {
         <Paragraph className={styles.description}>
           Je hebt een Git client nodig(optioneel), en je moet Node.js en NPM geïnstalleerd hebben. Dit gaat poort:8000
           gebruiken, dus zorg er voor dit poort niet al in gebruik is.
-          <CodeBlock
-            codeBlock={
-              <div>
-                $ cd /pwa <br />
-                $ npm install <br />
-                $ npm run develop <br />
-              </div>
-            }
-          />
+          <CodeBlock className={styles.codeblock}>
+            {`$ cd /pwa
+$ npm install
+$ npm run develop`}
+          </CodeBlock>
```

Comment:
@lencodes That solution wouldn't work great without CSS, since `span` elements are inline elements and they don't preserve newlines. How about using the [`dedent` npm package](https://www.npmjs.com/package/dedent) for this? We have had good experiences with that in Storybook code documentation.

For example:


```tsx
          <CodeBlock className={styles.codeblock}>
            {dedent`
            $ cd /pwa
            $ npm install
            $ npm run develop
          `}
          </CodeBlock>
```